src/qt/bitcoingui.cpp

```diff
@@ -251,28 +252,28 @@ void BitcoinGUI::createActions()
     overviewAction->setStatusTip(tr("Show general overview of wallet"));
     overviewAction->setToolTip(overviewAction->statusTip());
     overviewAction->setCheckable(true);
-    overviewAction->setShortcut(QKeySequence(Qt::ALT + Qt::Key_1));
+    overviewAction->setShortcut(QKeySequence(tr("Alt+1")));
```

Comment:
Isn't `tr` used to mark messages for translation? If yes, why is this needed here? If this is needed to convert to `QString`, why not call `QString{}` instead?

Reply:
> Isn't `tr` used to mark messages for translation?

Yes, it is.

> If yes, why is this needed here? If this is needed to convert to `QString`, why not call `QString{}` instead?

No need for translation here. Updated in the recent push.

Thanks!